tests/microos/toolbox.pm

```diff
@@ -60,7 +60,7 @@ sub run {
     select_console 'root-console';
     $self->create_user;
 
-    my $toolbox_image_to_test = get_var('CONTAINER_IMAGE_TO_TEST');
+    my $toolbox_image_to_test = get_var('CONTAINER_IMAGE_TO_TEST_EXTRA');
```

Comment:
Why this got renamed?

Reply:
I realized that, in this module for toolbox tests, when needed to update the default uri, we use that same parameter also used in other containers modules where logic is different, like image_podman, where failed in a VR.

In fact, in the the loop above, `tests/containers/image.pm:L#84`, calling [get_image_uri](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/633f61d452c908689aea88219bbb6b48c1440a17/lib/containers/urls.pm#L206), it will soon return that assigned url, instead of continuing on distri/version found in `CONTAINER_IMAGE_VERSIONS`. That url is not what we expect in containers tests.

Reply:
VR 1 above: 
[Here in toolbox](https://openqa.suse.de/tests/15649782#step/toolbox/12) we use `..._EXTRA`;
[here in image_podman](https://openqa.suse.de/tests/15649782#step/image_podman/115), using the standard parameter, the loop can inspect the containers in CONTAINER_IMAGE_VERSIONS: but still we can set eventual `...TO_TEST` for that logic, nothing to do with toolbox.

This happens in particular when HDD_ and elements in CONTAINER_IMAGE_VERSIONS have _diffrent distros_, like for SLE-M and SLES.

Reply:
This variable has been context-sensitive (i.e. test-run sensitive) since years and it worked just fine. The name `CONTAINER_IMAGE_TO_TEST` is descriptive to what it is, it defines the container under test.

We also don't use different `ISO` or `HDD_` variables depending on the test run, this is a more generic setting that served us well in the past.

***

When changing something this fundamental it needs to be well motivated. So please @m-dati, either make your case or please let's not rename something that is used all over the place since a long time without the need for doing so.

Reply:
Please, before read carfully the description provided in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20383#discussion_r1797118768.

I am not renaming anything, but **using another parameter** to address the local toolbox need. The original parameter CONTAINER_IMAGE_TO_TEST scope and usage reamain the same.

I simply realized that the `image explicit definition` in this toolbox module also affecting another next module, where the original CONTAINER_IMAGE_TO_TEST used for a totally different meaning and scope.

Therefore, this means that this for toolbox is a **totally different parameter**.
I named it _similar to the original_ simply adding a variation (_EXTRA), to mantain its origin, but to generalize it for further usage, but we can opt for other naming, like e.g. TOOLBOX_IMAGE_TO_TEST

Reply:
This point requires a discussion, let's reserve a time slot on Thursday for it.